tests/spec/time-flags.spec.js

```diff
@@ -1,14 +1,16 @@
-var Filer = require('../../src');
-var util = require('../lib/test-utils.js');
-var expect = require('chai').expect;
+'use strict';
+
+const Filer = require('../../src');
+const util = require('../lib/test-utils.js');
+const expect = require('chai').expect;
 
 describe('node times (atime, mtime, ctime) with mount flags', function() {
 
-  var dirname = '/dir';
-  var filename = '/dir/file';
+  let dirname = '/dir';
```

Comment:
All these `let` keywords can be switched to `const`, since the variables aren't going to change value.